appveyor.yml

```diff
@@ -1,25 +1,32 @@
 os: unstable
-cache:
-  - node_modules
+
 environment:
   matrix:
     - nodejs_version: 6
     - nodejs_version: 4
-install:
-  - ps: Install-Product node $env:nodejs_version
-  - set CI=true
-  - npm install -g npm@latest
-  - set PATH=%APPDATA%\npm;%PATH%
-  - npm install
+
+cache:
+  - node_modules
+
 matrix:
   fast_finish: true
+
 build: off
+
 version: '{build}'
+
 shallow_clone: true
+
 clone_depth: 1
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - set CI=true
+  - npm install -g yarnpkg
+  - yarn
+
 test_script:
   - node --version
-  - npm --version
```

Comment:
I would leave this line, as npm is still being used in places. I would also add `- yarn --version` below it.